superset/assets/visualizations/deckgl/layers/arc.jsx

```diff
@@ -8,6 +8,15 @@ import DeckGLContainer from './../DeckGLContainer';
 import * as common from './common';
 import sandboxedEval from '../../../javascripts/modules/sandbox';
 
+function getPoints(data) {
```

Comment:
Definitely think we should work this `getPoints()` into a deckGL utils folder. I see being used in each layer. I definitely think we should try and consolidate this method to work for all layers.

Reply:
The idea is that it can be different for each layer. In places where it's common across layers it's a simple accessor (`d => d.position`) which I'd rather keep as is than having a redirection to a `positionAccessor` helper in another file.

Reply:
Thinking about it now you are right it would be more work to actual implement the `positionAccessor` helper.